isce2_topsapp/topsapp_proc.py

```diff
@@ -25,6 +25,15 @@
 
 TEMPLATE_DIR = Path(__file__).parent/'templates'
 
+GEOCODE_LIST_BASE = ['merged/phsig.cor',
```

Comment:
We should check if we use all of these? I would only geocode the ones that gets used in the product. which would be fitlered coherence and filtered unwrapped phase corrected for DEM and topophase.

Reply:
These are taken from here: https://github.com/aria-jpl/ariamh/blob/develop/interferogram/sentinel/topsApp.xml.tmpl#L32